src/gsi/cplr_wrwrfmassa.f90

```diff
@@ -1850,6 +1850,8 @@ subroutine wrwrfmassa_netcdf_wrf(this,mype)
   !                        from sensible to potentionl temperature for writing
   !   2022-03-15  Hu  add a check to remove the negative moisture if rh>4%
   !                   for the lowest 3 model level
+  !   2025-01-28  zhao   - added code to write out analyses of wave height (howv) & wind gust (gust)
```

Comment:
Following the suggestion from Russ, this new changelog entry is not necessary and will be removed in the updated code.